usage/usageplot.py

```diff
@@ -69,11 +72,16 @@ def get_args(args):
         name = 'usage'
     else:
         name = args.name
+    
+    if args.color:
+        color = True
+    else:
+        color = False
```

Comment:
This conditions is redundant, `args.color` is already a flag (it is `True` when script is called with `-c` option and `False` when not).